dc_logging_client/log_entries.py

```diff
@@ -29,7 +29,13 @@ def as_log_line(self, newline=True):
         newline_char = ""
         if newline:
             newline_char = "\n"
-        json_data = json.dumps(self.__dict__, sort_keys=True, default=str)
+
+        data = self.__dict__
+        for k, v in data.items():
+            if isinstance(v, datetime.datetime):
+                data[k] = v.strftime("%Y-%m-%d %H:%M:%S.%f")[0:-3]
```

Comment:
Can you explain (in a comment for future us) why we need to truncate the 6 digit microseconds?